code/game/objects/effects/decals/crayon.dm

```diff
@@ -47,3 +47,289 @@ GLOBAL_LIST(gang_tags)
 /obj/effect/decal/cleanable/crayon/gang/Destroy()
 	LAZYREMOVE(GLOB.gang_tags, src)
 	..()
+
+// Mapping
+
+/obj/effect/decal/cleanable/crayon/rune
```

Comment:
having distinct subtypes for simple variations of things could be cluttered, you could potentially just set a mapload initialization proc for the basetype, and have checks to see what var is put, like a var/variation on the base type, then in sdmm you can set this var to 1-6 (for runes for example) and then on mapload it reads the set var and loads the icon state associated with that
	
```
/obj/effect/decal/cleanable/crayon/rune
	name = "Rune"
	icon_state = "rune1"
	var/variation = 1

/obj/effect/decal/cleanable/crayon/rune/Initialization(Mapload)
	. = ..()
	switch(variation)
		if(1)
			icon_state = "rune1"
		if(2)
			icon_state = "rune2"
		if(3)
			icon_state = "rune3"
		if(4)
			icon_state = "rune4"
		if(5)
			icon_state = "rune5"
		if(6)
			icon_state = "rune6"
```

this means by default the item when placed will place rune1, but if you change the var in sdmm it will actually be a different rune. of course this means it cant be previewed in sdmm to see how the rune looks but you can replace the numbers with a more descriptive name. doing this for all the type path lists here will massively reduce the amount of typepathes we need to sift through to grab something

Reply:
I gutted all the rune subtypes and stuck the icon states into a list with 877b512feade80201d72909aa6dcbca8ec914dc5